src/_includes/partials/footer.njk

```diff
@@ -1,40 +1,56 @@
 
 <footer id="wb-info">
-	<div class="landscape">
-		<nav class="container wb-navcurr">
-			<h2 class="wb-inv">{{ footer[locale].aboutGovernment }}</h2>
-			<ul class="list-unstyled colcount-sm-2 colcount-md-3">
-				<li><a href="{{ footer[locale].contactUsURL }}">{{ footer[locale].contactUs }}</a></li>
-				<li><a href="{{ footer[locale].departmentsAndAgenciesURL }}">{{ footer[locale].departmentsAndAgencies }}</a></li>
-				<li><a href="{{ footer[locale].publicServiceAndMilitaryURL }}">{{ footer[locale].publicServiceAndMilitary }}</a></li>
-				<li><a href="{{ footer[locale].newsURL }}">{{ footer[locale].news }}</a></li>
-				<li><a href="{{ footer[locale].treatiesLawsAndRegulationsURL }}">{{ footer[locale].treatiesLawsAndRegulations }}</a></li>
-				<li><a href="{{ footer[locale].governmentWideReportingURL }}">{{ footer[locale].governmentWideReporting }}</a></li>
-				<li><a href="{{ footer[locale].primeMinisterURL }}">{{ footer[locale].primeMinister }}</a></li>
-				<li><a href="{{ footer[locale].howGovernmentWorksURL }}">{{ footer[locale].howGovernmentWorks }}</a></li>
-				<li><a href="{{ footer[locale].openGovernmentURL }}">{{ footer[locale].openGovernment }}</a></li>
-			</ul>
-		</nav>
+	<h2 class="wb-inv">{{ footer[locale].aboutThisSite }}</h2>
+	<div class="gc-contextual">
+	<div class="container">
+			<nav>
+				<h3>{{ footer[locale].contextualHeader }}</h3>
+				<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3"><li><a href="http://canada.ca/en">{{ footer[locale].contextualLinkText1 }}</a></li><li><a href="{{ footer[locale].contextualLink2 }}"> {{ footer[locale].contextualLinkText2 }} </a></li><li><a href="http://canada.ca/en">{{ footer[locale].contextualLinkText3 }}</a></li></ul>
+			</nav>
+		</div>
 	</div>
```

Comment:
Remove contextual (DAT section) until pages have been created.

Reply:
Added the changes in the latest commit.